lib/on_membership_event.js

```diff
@@ -23,9 +23,20 @@ var Member = require('./membership/member.js');
 var MembershipEvents = require('./membership/events.js');
 
 function createChecksumComputedHandler(ringpop) {
-    return function onMembershipChecksumComputed() {
+    var membershipLogger = ringpop.loggerFactory.getLogger('membership');
+
+    return function onMembershipChecksumComputed(event) {
         ringpop.stat('increment', 'membership.checksum-computed');
         ringpop.emit('membershipChecksumComputed');
+
+        if (event.oldChecksum !== event.checksum) {
```

Comment:
would it make sense to also `stat` an `increment` in here so we can graph the useless computations?